sentry/templates/_helper.tpl

```diff
@@ -12,6 +12,22 @@
 {{- define "sentry.port" -}}9000{{- end -}}
 {{- define "snuba.port" -}}1218{{- end -}}
 
+{{- define "relay.image" -}}
+{{- default "getsentry/relay" .Values.images.relay.repository -}}
+:
+{{- default .Chart.AppVersion .Values.images.relay.tag -}}
+{{- end -}}
+{{- define "sentry.image" -}}
+{{- default "getsentry/sentry" .Values.images.sentry.repository -}}
+:
+{{- default .Chart.AppVersion .Values.images.sentry.tag -}}
+{{- end -}}
+{{- define "snuba.image" -}}
+{{- default "getsentry/snuba" .Values.images.snuba.repository -}}
+:
+{{- default .Chart.AppVersion .Values.images.snuba.tag -}}
```

Comment:
Shouldn't this be a `tpl` with the context passed in? `tpl .Chart.AppVersion .`?

Reply:
It seems to work as is.

```console
$ helm install --dry-run --generate-name --debug . | grep "getsentry\\/"
install.go:159: [debug] Original chart version: ""
install.go:176: [debug] CHART PATH: /home/yoan/soft/charts/sentry

        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
          image: "getsentry/relay:20.7.2"
        image: "getsentry/relay:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
```

Reply:
Interesting! I also gave bad code haha, but good to know.